ci/scripts/e2e-source-test.sh

```diff
@@ -53,6 +53,12 @@ if [ "$profile" == "ci-dev" ]; then
 fi
 risedev slt './e2e_test/source_inline/**/*.slt' -j4
 risedev slt './e2e_test/source_inline/**/*.slt.serial'
+
+if [ "$profile" == "ci-release" ]; then
```

Comment:
why this is release only? :eyes: (Better to add some comments

Reply:
fixed https://github.com/risingwavelabs/risingwave/pull/22199/commits/909444cb6254140f3e3d2ef8464517a2f0912a5b